CHANGELOG.txt

```diff
@@ -2,6 +2,8 @@ master:
  - obspy.core:
    * Casting FDSN identifiers to strings upon setting in the stats dictionary
      (see #1997).
+   * UTCDateTime objects will now always evaluate equal if their string
+     representations are equal.
```

Comment:
Please add the issue number (but I guess you'll do that anyway in a subsequent commit).